sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -1,30 +1,37 @@
 # Release History
 
-## 1.0.0-preview.3 (2019-11-04)
+## 5.0.0-preview.6
 
 ### Acknowledgments 
 
-Thank you to our developer community members who helped to make the Azure SDKs better with their contributions to this release:
+Thank you to our developer community members who helped to make the Event Hubs client libraries better with their contributions to this release:
 
-- Vyacheslav Benedichuk _([GitHub](https://github.com/vbenedichuk))_
+- Christopher Scott _([GitHub](https://github.com/christothes))_
 
 ### Changes
 
 #### Organization and naming
 
-- For consistency with the official nomenclature of the Azure Storage Blobs service, the package has been renamed to `Azure.Messaging.EventHubs.CheckpointStore.Blobs` and the namespaces have also been adjusted to "Blobs."  
-  _(A community contribution, courtesy of [vbenedichuk](https://github.com/vbenedichuk))_
+- This version is a reorganization of the Event Hubs client library packages, moving the `EventProcessorClient` into its own package and evolving this version into an opinionated implementation on top of Azure Storage Blobs.  This is intended to offer a more streamlined developer experience for the majority of scenarios and allow developers to more easily take advantage of the processor.
 
-- Some minor documentation updates to reflect the changes made to the `EventProcessorClient` and its associated namespaces.
+- A large portion of the public API surface, including members and parameters, have had adjustments to their naming in order to improve discoverability, provide better context to developers, and better conform to the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html).
 
-## 1.0.0-preview.2 (2019-10-09)
+#### Event Processing
```

Comment:
Should we include error handling, retry policy and immediate task cancellation via `CancellationToken` in the `EventProcessorClient` and `BlobsCheckpointStore` types?

Reply:
Technically we had error handling exposed previously....we just didn't have it internally wired up.   I don't know that we need to go to the level of calling out that we rounded out cancellation support, unless you know of issues/feedback that were asking for it.   If so, please let me know and I'll amend in another set of changes.

Reply:
Nah, I'm just overthinking. I wasn't sure if we should include them exactly because they are all internal changes. I think we are good.